sql/rikai--0.1.sql

```diff
@@ -26,6 +26,27 @@ AS $$
 	return rikai.__version__.version
 $$ LANGUAGE plpython3u;
 
+CREATE FUNCTION ml.python_version()
```

Comment:
nits, should we just make one `ml.version` that returns 

```
| rikai version | ... |
| python version | ... |
| rikai-pg version | ... |
```

Similar to `numpy.show_info()` and such.

Reply:
`ml.show_info()`:

```
{"python": "3.8.10 (default, Mar 15 2022, 12:22:08) ", 
"rikai": "0.1.9",
 "torch": "1.11.0+cu102",
 "pythonpath": ["/usr/lib/python38.zip", "/usr/lib/python3.8", "/usr/lib/python3.8/lib-dynload", "/var/lib/postgresql/.local/lib/python3.8/site-packages", "/usr/local/lib/python3.8/dist-packages", "/usr/lib/python3/dist-packages"]}
```

Reply:
what is the returns value? jsonblob?

Reply:
yeah json blob